src/libs/PolicyUtils.js

```diff
@@ -166,6 +166,56 @@ function getIneligibleInvitees(policyMembers, personalDetails) {
     return memberEmailsToExclude;
 }
 
+/**
+ * Fetches the tag from policy tags, defaults to the first if no key is provided.
```

Comment:
From my understanding _fetch_ more suitable for some API work. The same comment for other cases. 
```suggestion
 * Gets the tag from policy tags, defaults to the first if no key is provided.
```